Dockerfile

```diff
@@ -1,19 +1,25 @@
+### This is a generated file from Dockerfile.in ###
+#@follow_tag(openshift-golang-builder:1.14)
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+
+ENV BUILD_VERSION=${CI_CONTAINER_VERSION}
+ENV OS_GIT_MAJOR=${CI_X_VERSION}
+ENV OS_GIT_MINOR=${CI_Y_VERSION}
+ENV OS_GIT_PATCH=${CI_Z_VERSION}
+ENV SOURCE_GIT_COMMIT=${CI_ELASTICSEARCH_OPERATOR_UPSTREAM_COMMIT}
+ENV SOURCE_GIT_URL=${CI_ELASTICSEARCH_OPERATOR_UPSTREAM_URL}
+
+
 WORKDIR /go/src/github.com/openshift/elasticsearch-operator
-COPY apis apis
-COPY controllers controllers
-COPY files files
-COPY internal internal
-COPY manifests manifests
-COPY vendor vendor
-COPY version version
-ADD Makefile main.go ./
-RUN go build -o bin/elasticsearch-operator main.go
+COPY ${REMOTE_SOURCE} .
```

Comment:
we were intentionally not copying everything before... can you please fix?

Reply:
Curious - which parts are we not copying and why do they need to be in the src/ directory if they are not needed to build?

Reply:
modified